Assets/Scripts/MenuScript.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MenuScript : MonoBehaviour
+{
+
+    public string firstLevel;
```

Comment:
We should always make private variables
This is not following the naming scheme please goto read the C# style guide in our repo's root folder